src/doc/help.py

```diff
@@ -339,3 +341,19 @@ def help_attribute_table():
     ]
     html = html_table(title, intro, inputs, outputs, more)
     return html
+
+    
+def help_export_kml():
+    title = tr('Export KML')
+    intro = tr('Export a vector layer as KML')
+    inputs = [
+        tr('Choose a vector layer')
+    ]
+    outputs = [
+        tr('Choose a path and name to store the KML file')
+    ]
+    more = [
+    ]
+    html = html_table(title, intro, inputs, outputs, more)
+    return html
+    
```

Comment:
Like here, there some spaces on this line. You need to remove them. It should an empty new line.